plugins/example/readme.md

```diff
@@ -0,0 +1,77 @@
+# example plugin
+
+## How to use
+
+### use plugin though Kubefile
```

Comment:
Is it possible to use the plugin in a more uniform way, and then in this way, there is no need to write its own usage for each plugin?